src/main/java/seedu/agendum/ui/TaskCard.java

```diff
@@ -38,6 +38,12 @@ public void initialize() {
         name.setText(task.getName().fullName);
         id.setText(displayedIndex + ". ");
         tags.setText(task.tagsString());
+        if (task.isCompleted()) {
+            name.setText(task.getName().fullName + " (done!)");
+            cardPane.setStyle("-fx-background-color: aquamarine");
+        } else {
+            cardPane.setStyle("-fx-background-color: salmon");
+        }
```

Comment:
Wouldn't a change to UI here in this PR cause a conflict with @fanwgwg's UI-update branch? I think it's better to just update models in this PR and update UI in an subsequent PR?